pages/common/7za.md

```diff
@@ -0,0 +1,19 @@
+# 7za
+
+> A file archiver with highest compression ratio
+
+- package and compress directory or file
+
+`7za a {{compressed.7z}} {{directory_or_file_to_compress}}`
+
+- decompress an existing 7z file
+
+`7za x {{compressed.7z}}`
+
+- compress to zip format
+
+`7za a -tzip {{compressed.zip}} {{directory_or_file_to_compress}}`
+
+- create multipart 7zip file; part_size: specifies part size in Bytes, Kilobytes, Megabytes or Gigabytes.
```

Comment:
the ":" is unnecessary here. Also maybe wrap "part_size" in code tags `` or the token syntax {{}} to make it explicit that it refers to the command parameter below.